services/category.js

```diff
@@ -29,6 +29,9 @@ const categoryService = {
       }
     ])
 
+    if (!minMaxPrices[0].min) minMaxPrices[0].min = 0
```

Comment:
can't it be done with something like $or inside $group?

Reply:
it would be really great and have been trying but there is no such operator $or or something similar in mongo aggregation to set default value